### DISCUSSION TITLE
Torque question

### DISCUSSION THREAD

**Discussion by doppleware** [QualiTorque/qualitorque.github.io#103] [Ideas]
<img width="701" alt="image" src="https://user-images.githubusercontent.com/93863/184231849-8ea76067-94b8-4eef-8b31-22127e9d2cfe.png">

It would be great to have an indication here that you can edit/add cloud accounts vis the admin menu.

**Comment by Shnitselon** [comment]
@doppleware Hi Roni,
Good to see you here. 

@TomerAdmon this is good Torque application feedback. Thoughts?

**Comment by shirabendor-quali** [comment]
Will handle

**Comment by shirabendor-quali** [comment]
Hey @doppleware ✨✨
I'm happy to share that we've just release a few enhancements around this area according to your feedback. Let us know what you think,.

Shira

**Comment by doppleware** [comment]
Awesome will check it out!
